### PR TITLE
GEODE-2594 Remove tools.jar and --pid options from docs

### PR DESCRIPTION
**Pull request by karensmolermiller**
@kirklund @davebarnes97 @joeymcallister Please review.

### CODE REVIEW

File: geode-docs/configuring/running/running_the_cacheserver.html.md.erb [hunk 1]
```diff
@@ -31,7 +31,7 @@ The Geode server is used primarily for hosting long-lived data regions and for r
 
 The `gfsh` utility uses a working directory for its configuration files and log files. These are the defaults and configuration options:
 
--   When you start a standalone server using `gfsh`, `gfsh` will automatically load the required JAR files `$GEMFIRE/lib/server-dependencies.jar` and `$JAVA_HOME/lib/tools.jar` into the CLASSPATH of the JVM process. If you start a standalone server using the ServerLauncher API, you must specify `$GEMFIRE/lib/server-dependencies.jar` inside your command to launch the process. For more information on CLASSPATH settings in Geode, see [Setting Up the CLASSPATH](../../getting_started/setup_classpath.html).
+-   When you start a standalone server using `gfsh`, `gfsh` will automatically load the required JAR file `$GEMFIRE/lib/server-dependencies.jar` into the CLASSPATH of the JVM process. If you start a standalone server using the ServerLauncher API, you must specify `$GEMFIRE/lib/server-dependencies.jar` inside your command to launch the process. For more information on CLASSPATH settings in Geode, see [Setting Up the CLASSPATH](../../getting_started/setup_classpath.html).
```
Comment (kirklund) [inline geode-docs/configuring/running/running_the_cacheserver.html.md.erb]: All references to $GEMFIRE/lib/server-dependencies.jar should change to $GEODE/lib/geode-dependencies.jar

File: geode-docs/configuring/running/running_the_locator.html.md.erb [hunk 2]
```diff
@@ -35,7 +35,7 @@ You can run the locator standalone or embedded within another Geode process. Run
...
 
--   When you start a standalone locator using `gfsh`, `gfsh` will automatically load the required JAR files (`$GEMFIRE/lib/locator-dependencies.jar`) into the CLASSPATH of the JVM process. If you start a standalone locator using the `LocatorLauncher` API, you must specify `$GEMFIRE/lib/locator-dependencies.jar` inside the command used to launch the locator process. For more information on CLASSPATH settings in Geode, see [CLASSPATH Settings for Geode Processes](../../getting_started/setup_classpath.html). You can modify the CLASSPATH by specifying the `--classpath` parameter.
+-   When you start a standalone locator using `gfsh`, `gfsh` will automatically load the required JAR file (`$GEMFIRE/lib/locator-dependencies.jar`) into the CLASSPATH of the JVM process. If you start a standalone locator using the `LocatorLauncher` API, you must specify `$GEMFIRE/lib/locator-dependencies.jar` inside the command used to launch the locator process. For more information on CLASSPATH settings in Geode, see [CLASSPATH Settings for Geode Processes](../../getting_started/setup_classpath.html). You can modify the CLASSPATH by specifying the `--classpath` parameter.
```
Comment (kirklund) [inline geode-docs/configuring/running/running_the_locator.html.md.erb]: All references to $GEMFIRE/lib/locator-dependencies.jar should change to $GEODE/lib/geode-dependencies.jar

File: geode-docs/configuring/running/running_the_cacheserver.html.md.erb (hunk 1 above)
Comment (karensmolermiller) [inline geode-docs/configuring/running/running_the_cacheserver.html.md.erb]: That task is being tracked with Geode-3208.

File: geode-docs/configuring/running/running_the_locator.html.md.erb (hunk 2 above)
Comment (karensmolermiller) [inline geode-docs/configuring/running/running_the_locator.html.md.erb]: That task is being tracked with Geode-3208.

**Pull request closed without merging**